backend/src/apiserver/server/pipeline_server.go

```diff
@@ -113,17 +113,14 @@ func (s *PipelineServer) createPipeline(ctx context.Context, pipeline *model.Pip
 	}
 	pipeline.Namespace = s.resourceManager.ReplaceNamespace(pipeline.Namespace)
```

Comment:
Similar to above, here we just replaces the namespace with empty string if we are not in multi-user mode. Should we report an error instead?

Reply:
I might be wrong on this, but see my opinion [here](https://github.com/kubeflow/pipelines/pull/9373/files#r1191532729)